### ISSUE TITLE
Time difference based on process.uptime()

### ISSUE THREAD

**Issue by Pupix** [braveg1rl/performance-now#6]
Hello!

Would be nice to have the time difference be based on `process.uptime()` so it acts more like the real `performance.now` on the first call.

With [electron](http://electron.atom.io) I was getting values all over the place because of this difference.

## The problem (exaggerate for the sake of it)
Before this *require* there could be `async code`, `legacy code` or just a `black box` to which you don't have access to *require* the library earlier.

```js
setTimeout(() => {
    let now = require("performance-now");
    console.log(now());
}, 2000);
```
The result should be `2000+ ms`

Currently that would log, on my machine `19.633549 ms`.

## The solution

Rewriting it as follows (sorry, no CoffeeScript) I've been able to get the expected output of `2000+ ms`

```js
const loadTime = process.hrtime();
const upTime = process.uptime();

// Get elapsed seconds
const seconds = Math.floor(upTime);
// Get the remaining milliseconds and cast to nanoseconds
const nanoseconds = (upTime - seconds) / 1e6;

loadTime[0] -= seconds;
loadTime[1] -= nanoseconds;

const getTime = () => {
    let hr = process.hrtime(loadTime);
    return (hr[0] * 1e9 + hr[1]) / 1e6;
};

module.exports = getTime;
```

All it actually does is remove the elapsed time between the `process.uptime` and the `process.hrtime` call. 

---

*N.B. The solution above does not take into consideration the `Date` fallbacks.*

Cheers,
Pupix

**Comment by braveg1rl** [comment]
Good call on using `process.uptime`. At the time I wrote this module, I wasn't aware of this function at all.

I've added some extra tests using isolated processes, in order to test the various scenarios in which this module could be used. 

I've just published version 2.0.0 which should do exactly what you want. Version 1.0 will keep functioning as before.

**Comment by Pupix** [comment]
Thanks!
